### PR TITLE
Adds New Tajaran Charm: Twin Suns

### PR DESCRIPTION
**Pull request by Ben10083**
New Twin Suns charm representing the suns of S'rend'marr, used by those who worhship S'rendarr and Messa

### CODE REVIEW

**Comment by Ben10083** [comment]
WIP until I can get a good extended description for it, also feedback for sprites encouraged

**Comment by Ben10083** [comment]
!wip

**Comment by Ben10083** [comment]
!review

File: code/modules/client/preference_setup/loadout/items/xeno/tajara.dm [hunk 1]
```diff
@@ -477,6 +477,7 @@
 	charm["bone charm"] = /obj/item/clothing/accessory/tajaran/charm/bone
 	charm["silver seashell charm"] = /obj/item/clothing/accessory/tajaran/charm/steel/silver/seashell
 	charm["tajani charm"] = /obj/item/clothing/accessory/tajaran/charm/tajani
+	charm["twin suns charm"] = 	/obj/item/clothing/accessory/tajaran/charm/twin_suns
```
Comment (DreamySkrell) [inline code/modules/client/preference_setup/loadout/items/xeno/tajara.dm:480]: excessive spaces

**Review by NonQueueingMatt — APPROVED** [review]

**Review by Geevies — APPROVED** [review]

**Pull request merged**